### ISSUE TITLE
AccountSelectorMulti doesn't work with screen reader

### ISSUE THREAD

**Issue by 42tte** [SpareBank1/designsystem#501]
#### Which package is this issue related to?
ffe-account-selector-react

#### Describe your issue (screenshots welcome!)
The accountselector does not work with screen reader (VO/Safari) and needs thorough testing and overhauling. Exemple of bugs:
* No information is given on that there is alternatives neighter are they read on focus.
* Not able to type in input field
* Add space event as selector?
* role=combobox is on the wrong element, should be on input
* 

#### What is the expected behavior? 
It should be usable with screen reader

#### What is the actual behavior?
It is not

#### Steps to reproduce
Use a screen reader on https://design.sparebank1.no/styleguidist/index.html#!/AccountSelectorMulti

**Comment by ivarni** [comment]
Instead of continuing to reinvent the wheel, should we just swap to using [downshift](https://github.com/paypal/downshift)?

**Comment by 42tte** [comment]
I second that, if no one else takes it upon them to test that the Downshift examples actually work I will try to do it asap (maybe next week).

First quick test of examples 01-04 shows it also have some issues, but generally it works better than ours and it would be a good thing contributing to it.
